packages/web-app-files/src/components/Upload/ProgressBar.vue

```diff
@@ -23,6 +23,7 @@
       <div class="uk-width-expand uk-text-truncate">
         <translate
           v-if="count === 1"
+          id="upload-progress-single"
```

Comment:
Please also add `files-` as a prefix to all the newly introduced ids. Kind of serves as a namespace of the app. Good that you already have a component namespace (`upload-progress`). 👍